src/patternfly/components/Login/examples/login-invalid-example.hbs

```diff
@@ -28,71 +27,59 @@
+            {{#> button button--modifier="pf-m-primary pf-m-block" button--attribute='type="submit"'}}
               Log in
             {{/button}}
-            {{#> check}}
-              {{#> check-input check-input--attribute='type="checkbox" id="invalid-login-demo-checkbox" name="invalid-login-demo-checkbox"'}}{{/check-input}}
-              {{#> check-label check-label--attribute='for="invalid-login-demo-checkbox"'}}Keep me logged in for 30 days.{{/check-label}}
-            {{/check}}
           {{/form-group}}
         {{/form}}
       {{/login-main-body}}
       {{#> login-main-footer}}
-        {{#> title titleType="h2" title--modifier="pf-m-md"}}
-          Or log in with
-        {{/title}}
-        {{#> login-main-footer-grid}}
-          {{#> login-main-footer-grid-item}}
-            {{#> login-main-footer-link}}
-              <img src="/assets/images/logo-google.svg" alt="">
-              {{#> login-main-footer-link-text}}
-                Google
-              {{/login-main-footer-link-text}}
-            {{/login-main-footer-link}}
-          {{/login-main-footer-grid-item}}
-          {{#> login-main-footer-grid-item}}
-            {{#> login-main-footer-link}}
-              <img src="/assets/images/logo-github.svg" alt="">
-              {{#> login-main-footer-link-text}}
-                GitHub
-              {{/login-main-footer-link-text}}
-            {{/login-main-footer-link}}
-          {{/login-main-footer-grid-item}}
-          {{#> login-main-footer-grid-item}}
-            {{#> login-main-footer-link}}
-              <img src="/assets/images/logo-dropbox.svg" alt="">
-              {{#> login-main-footer-link-text}}
-                Dropbox
-              {{/login-main-footer-link-text}}
-            {{/login-main-footer-link}}
-          {{/login-main-footer-grid-item}}
-          {{#> login-main-footer-grid-item}}
-            {{#> login-main-footer-link}}
-              <img src="/assets/images/logo-facebook.svg" alt="">
-              {{#> login-main-footer-link-text}}
-                Facebook
-              {{/login-main-footer-link-text}}
-            {{/login-main-footer-link}}
-          {{/login-main-footer-grid-item}}
-          {{#> login-main-footer-grid-item}}
-            {{#> login-main-footer-link}}
-              <img src="/assets/images/logo-gitlab.svg" alt="">
-              {{#> login-main-footer-link-text}}
-                GitLab
-              {{/login-main-footer-link-text}}
-            {{/login-main-footer-link}}
-          {{/login-main-footer-grid-item}}
-          {{#> login-main-footer-grid-item login-main-footer-grid-item--modifier="pf-m-more"}}
-            {{#> button button--attribute='id="login-demo-more"' button--modifier="pf-m-link"}}
-              More...
-            {{/button}}
-          {{/login-main-footer-grid-item}}
-        {{/login-main-footer-grid}}
+        {{#> login-main-footer-links}}
```

Comment:
The naming of this seems to assume that there will only ever be links in the footer, just checking on this?

Reply:
This should always be a list of links. So I think Links is ok but don't feel strongly about it. @mcoker what do you think?